### DISCUSSION TITLE
2024/06/29/memtier

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#133] [Q&A]
# 2024/06/29/memtier

https://wanghenshui.github.io/2024/06/29/memtier.html

**Comment by suxb201** [comment]
需求时要同时给一批实例加压吗，为什么不能简单写个 shell 脚本实现呢

**Reply by wanghenshui** [reply]
脚本是同时的，不是这种round robin轮流模式的，不过脚本也可以实现
